src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User user = new UserService().findByEmail(email);
+        if ((user != null) && (user.getPassword() != null) && (user.getPassword().equals(password))) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
this check will suffice
```suggestion
        if (user != null) {
            return user.getPassword().equals(password);
        }
        return false;
```